### DISCUSSION TITLE
Subtitles not displayed by default in 23.12

### DISCUSSION THREAD

**Discussion by wssddc** [smplayer-dev/smplayer#873] [General]
Using version 23.12, I don't see (embedded) subtitles for .mkv files unless I specifically ask for them from the menu.  Same videos using 23.6 show the subs automatically.  Multimedia engine is set to mpv (only one offered on some of my PCs).  OS is Windows 10 22H2.

**Comment by smplayer-dev** [comment]
I think I didn't change anything regarding subtitle selection since version 23.06. 
Anyway go to preferences -> general -> preferred audio and subtitles and change the subtitle track to 1. That should select the first subtitle track.

**Reply by wssddc** [reply]
Yes, that mostly fixes it.  But if the language of the first subtitle track does not match the configured preferred language, and track 2 does match, I get track 1.  In practice, I don't know if I have any .mkv files that trigger this.  I had to edit headers with mkvtoolnix gui to generate a test case.  This is also true with 23.06.  So perhaps nothing even worth looking at.  Thanks.

**Reply by OldGeezer916** [reply]
Yeah, that fixes it for me. Smplayer is the best player, but SO many defaults you have to change. My older version either didn't have this choice or it was always selected.

**Comment by Xenos71** [comment]
Hi, I downloaded portable version 23.12 and I also noticed that the embedded subtitles of some mkv files are not being displayed, although they were flaged as the default subtitle track.
I played the same mkv files with the previous version (23.6) and the subtitles are being displayed properly.  
After some tests, I realized that the default subtitles are being ignored only in mkv files with 6-channel audio. Strange, but maybe it can help you troubleshoot this problem.
Thanks in advance.

**Reply by Raza-0007** [reply]
Try upgrading mpv to version 0.37, and see if it resolves your issue. 

smplayer 23.12 has mpv 0.36 while smplayer 23.6 had mpv 0.35. They changed something in mpv 0.36 related to subtitle selection that did not work, so it was reverted back in 0.37. See the 0.37 [release notes](https://github.com/mpv-player/mpv/releases) under the sub heading "Changed". 

>options: restore old default subtitle selection behavior 

~Raza

**Reply by Xenos71** [reply]
Yes! Using mpv 0.37 fixed the issue.
Thank you so much Raza!
